tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -4265,6 +4346,11 @@ class HLSLExternalSource : public ExternalSemaSource {
         DXASSERT(decl->isImplicit(),
                  "otherwise object template decl is not set to implicit");
         return AR_TOBJ_OBJECT;
+      } else if (auto *TAT = dyn_cast<TypeAliasTemplateDecl>(typeRecordDecl)) {
+        if (TAT->getCanonicalDecl() == +m_defaultMatDecl->getCanonicalDecl() ||
```

Comment:
What's with the `+` in `+m_defaultMatDecl->getCanonicalDecl()` and on the next line?  Was this a copy-paste from a diff with clang-format on paste incorporating the plus into the expression?

Reply:
Good catch. Fixed.